crates/superposition/src/main.rs

```diff
@@ -83,6 +84,25 @@ async fn main() -> Result<()> {
             .split(',')
             .map(String::from)
             .collect::<HashSet<String>>();
+    let mandatory_dimensions: Map<String, Value> =
+        get_from_env_unsafe::<String>("MANDATORY_DIMENSIONS")
+            .expect("MANDATORY_DIMENSIONS is not set")
+            .split(';')
```

Comment:
better to create this part a function and write tests for it since we are doing 3 times split with using array index

Reply:
This solution is not permanent, will get removed afterwards when tenant config gets in.
I think this can go right now.